portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 1;
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
```

Comment:
prefer 'index' over 'i'. More descriptive names preferred in general

https://google.github.io/styleguide/jsguide.html#naming-rules-common-to-all-identifiers

Reply:
done!